wagtailaltgenerator/providers/rekognition_service.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+import boto3
+import requests
+from django.conf import settings
+from wagtailaltgenerator.providers import AbstractProvider, DescriptionResult
+
+
+class RekognitionService(AbstractProvider):
+
+    def __init__(self, *args, **kwargs):
+        self.client = boto3.client('rekognition')
+        super(RekognitionService, self).__init__(*args, **kwargs)
+
+    def describe(self, image):
+        image_url = image.file.url
+        image_data = requests.get(image_url)
+
+        description = None
+        tags = []
+
+        if image_data.status_code == 200:
+            response = self.client.detect_labels(
+                Image={
+                    'Bytes': image_data.content,
+                },
+                MinConfidence=50
```

Comment:
Good idea! I will check if it's possible to translate this setting over to the cognitive provider.